support/donations.md

```diff
@@ -0,0 +1,32 @@
+---
+breadcrumb: Donations
+---
+# Sponsorship and Donations
+
+The OpenSSL project relies on funding to maintain and improve OpenSSL.
+
+You can support the OpenSSL project financially with the purchase of a
+[support contract](contracts.html), or by a sponsorship donation.
+
+We can accept smaller sponsorship donations via
+[GitHub Sponsors](https://github.com/sponsors/openssl).
+
+We do not have a PayPal account. Please do not donate to any PayPal
+account claiming to be associated with us!
+
+Please note that the [OpenSSL Software Foundation](/community/contacts.html)
+(OSF) is incorporated in the the state of Delaware, United States, as a
+non-profit corporation. It does not qualify as a tax-exempt charitable
+organisation under Section 501(c)(3) of the U.S. Internal Revenue Code.
+
+We provide [Acknowledgements for sponsors](acks.html) depending on the level
+of funding:
+
+|                               |                                                                   |
+|-------------------------------|-------------------------------------------------------------------|
```

Comment:
This is the documented way to get a header-less table.  From https://pandoc.org/MANUAL.html#extension-pipe_tables:

> ... The header cannot be omitted. To simulate a headerless table, include a header with blank cells.

Reply:
An alternative would be to alter our CSS to put a bottom line on \<thead> instead of tr:first-child.

Quite frankly, I'd rather do that...  thoughts?

Reply:
... actually, it's perfectly possible to make it work both for `table > tr:first-child` and `table > thead`

Reply:
I backed away from this...  See the new commit in #350, which adds the CSS change I proposed.  It is live on https://beta.openssl.org, and appears to do things right for a table with \<thead> and \<tbody>